scripts/test-python-artifacts.py

```
@@ -0,0 +1,174 @@
+            lambda d: d.update({f'kisnet_ytm-{a.VERSION}.dist-info/licenses/LICENSE.md': b'stale license'}),
+            lambda d: d.update({f'kisnet_ytm-{a.VERSION}.dist-info/METADATA': b'Name: wrong\nVersion: 0.2.0\n'}),
+        ]
+        for mutate in mutations:
+            with self.subTest(mutate=mutate):
+                path = candidate(self.directory, self.target, mutate)
+                with self.assertRaises(ValueError):
+                    a.inspect(path, self.target)
+
+    def test_unshipped_native_dependencies(self):
+        for target in a.POLICY['targets']:
+            def mutate(data):
```

Ruff (0.16.3)

[warning] Missing return type annotation for private function `mutate`

Add return type annotation: `None`

(ANN202)